pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: messenger
-version: 0.2.0
+version: 0.2.1
```

Comment:
Due to App Store rules, the applicaiton's version must always be incremented after the latest approved release. The latest `main` CI has failed to upload the application to TestFlight exactly because of this:

`The value for key CFBundleShortVersionString [0.2.0] in the Info.plist file must contain a higher version than that of the previously approved version [0.2.0].`